test/evergreen.yml

```diff
@@ -3496,7 +3496,7 @@ tasks:
           script: |
             set -o errexit
             set -o verbose
-            export "PATH=/opt/mongodbtoolchain/v4/bin:$PATH"
```

Comment:
Instead of the export, we should have been using the `PREPARE_TEST_ENV` expansion instead. I am happy to merge this for now, please ensure that we use `PREPARE_TEST_ENV` when officially move all the tests

Reply:
sgtm

Reply:
Yep I agree we should be using `PREPARE_TEST_ENV` I'll add a note in [WT-14127](https://jira.mongodb.org/browse/WT-14127) to remind us to make this change.